.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 89
```

Comment:
I think that `black` by default has a `88` line.
Docs: https://black.readthedocs.io/en/stable/the_black_code_style.html
> If you’re using Flake8, you can bump max-line-length to 88 and forget about it.